src/__tests__/rest/compliance/nfts/base.ts

```diff
@@ -0,0 +1,157 @@
+import { assertTagPresent } from '~/assertions';
+import { TestFactory } from '~/helpers';
+import { RestClient } from '~/rest';
+import { ProcessMode, TxBase } from '~/rest/common';
+import {
+  bothConditionsRequirements,
+  complianceRequirementParams,
+  complianceRequirementsParams,
+  receiverConditionsRequirements,
+  senderConditionsRequirements,
+} from '~/rest/compliance';
+import { Identity } from '~/rest/identities/interfaces';
+import { createNftCollectionParams } from '~/rest/nfts';
+
+const handles = ['issuer', 'blocked', 'investor'];
+
+let factory: TestFactory;
+
+describe('Compliance Requirements for NFTs', () => {
+  let restClient: RestClient;
+  let signer: string;
+  let issuer: Identity;
+  let blocked: Identity;
+  let investor: Identity;
+  let ticker: string;
+  let signerTxBase: TxBase;
+
+  beforeAll(async () => {
+    factory = await TestFactory.create({ handles });
+    ({ restClient } = factory);
+    issuer = factory.getSignerIdentity(handles[0]);
+    blocked = factory.getSignerIdentity(handles[1]);
+    investor = factory.getSignerIdentity(handles[2]);
+
+    ticker = factory.nextTicker();
+    signer = issuer.signer;
+    signerTxBase = { options: { signer, processMode: ProcessMode.Submit } };
+
+    const nftParams = createNftCollectionParams(
+      ticker,
+      [{ type: 'Local', name: 'Test', spec: { description: 'test metadata' } }],
+      signerTxBase
+    );
+    await restClient.nfts.createNftCollection(nftParams);
+  });
+
+  afterAll(async () => {
+    await factory.close();
+  });
+
+  it('method: getComplianceRequirements', async () => {
+    const result = await restClient.compliance.getComplianceRequirements(ticker);
+
+    expect(result).toMatchObject({
+      requirements: expect.any(Array),
+    });
+  });
+
+  it('method: setRequirements', async () => {
```

Comment:
```suggestion
  it('should set requirements', async () => {
```

(applies to each case in this file)